sql/core/src/test/resources/sql-tests/inputs/array.sql

```diff
@@ -92,13 +92,31 @@ select
 from primitive_arrays;
 
 -- index out of range for array elements
+-- return null results if array index in [] operator is out of bound
+set spark.sql.ansi.strictIndexOperator=false;
```

Comment:
since it's sub config of ANSI mode, shall we move the new tests to `.../ansi/array.sql`?

Reply:
sure, updated